### DISCUSSION TITLE
Copy Anchor Links to Clipboard

### DISCUSSION THREAD

**Discussion by caseyamp** [squidfunk/mkdocs-material#4297] [Q&A] [answered]
Howdy, we use the [anchor tracking feature](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/?h=anchor#anchor-tracking). Our readers have repeatedly requested that we make it so clicking the anchor link icon copies the full URL to the user's clipboard. 

Is there a quick way to implement this functionality without having to do full-on theme development?

**Comment by squidfunk** [comment, marked as answer]
Sure, add something like this as additional JavaScript (should also work with instant loading):

``` js
document$.subscribe(() => {
  const links = Array.from(document.querySelectorAll(".headerlink"))
  for (const link of links) {
    link.setAttribute("data-clipboard-text", link.href)
  }
})
```

**Reply by caseyamp** [reply]
Wow, I was definitely overthinking it. Thank you @squidfunk it's working perfectly.
